CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/planar/PQTree.java

```diff
@@ -113,24 +113,7 @@ public void reduce() {
                 if (node.getParent().getPertinentChildCount() == 0) {
                     nodesToProcess.addLast(node.getParent());
                 }
-                boolean matched
```

Comment:
Did this variable and the related one in the else clause have a use at some stage? Just observing that removing these variables means the functions that created these booleans (>200 lines of code) are also now unused.